### PR TITLE
Contributing guide

### PR DESCRIPTION
**Pull request by hobbes7878**
A first draft. Whatchu think? Reasonable?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Contributions are always welcome from members of the Reuters Graphics team.
+
+If anyone outside our team ever starts using these components, we will happily review any PRs or issues, **BUT** if they are designed solely to benefit a use case that isn't ours, they likely won't be merged.
+
+## How should Reuters Graphics staff contribute?
+
+Code contributions should be made through the normal [GitHub Flow](https://www.w3schools.com/git/git_github_flow.asp#:~:text=The%20GitHub%20flow%20is%20a,Make%20changes%20and%20add%20Commits). Basically, make a branch and submit a pull request (PR) for any new features or changes.
+
+Before you make a PR, we recommend you create an issue on this repo describing what is missing, broken or could be added or improved. (We'll close that issue when we merge your PR.)
```
Comment (juruwolfe) [inline CONTRIBUTING.md:21]: Might make sense to lead with this and next graf so we're following the chronology we want them to follow: First make issue, then branch, code, PR if that makes sense

**Review by juruwolfe — APPROVED** [review]
👍 This looks good to me but left a suggestion and one thought.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+It's helpful if that issue describes what changes you propose to make at a high level so we can agree on a general direction before you write code. That's especially true if code you're writing will change how others need to write theirs. If needed, provide any links to best practice guidelines that support the change you want to make. Tag others on the team who may contribute to any discussion needed. Always tag an editor.
+
+Once you're ready to submit code, be sure it's properly formatted before you ask for a review. The easiest way is to ensure the built-in code formatters are working. (They should.) Be sure to add comments around any tricky bits of logic you're adding. Even better, document your code using [JSDoc](https://devhints.io/jsdoc) comments. (Check out [JSDoc shortcuts](https://code.visualstudio.com/docs/languages/javascript#_jsdoc-support) in VS Code for a leg up.)
+
+Once you've submitted your PR, tag an editor to review it.
+
+An editor will approve your PR after addressing any issues they see. Once an editor approves and there are no code conflicts, you can merge your PR into master.
```
[the remark this answers is not in the document]
Comment (juruwolfe) [reply CONTRIBUTING.md:31]: Yeah I would love to be able to help with this stuff. Maybe we can find some time (during a Monday chat?) to discuss overall guiding philosophy, etc.

[the remark this answers is not in the document]
Comment (juruwolfe) [reply CONTRIBUTING.md:31]: But if Fielding is game, that would be ideal as well

[the remark this answers is not in the document]
Comment (hobbes7878) [reply CONTRIBUTING.md:31]: Definitely, let's have that chat!

**Pull request merged**